reliability/single-fit/trace/build.gradle.kts

```diff
@@ -32,6 +32,10 @@ android {
         // Required because AGP doesn't support kotlin test fixtures :/
         java.srcDir("${project.rootDir.path}/dd-sdk-android-core/src/testFixtures/kotlin")
     }
+
+    compileOptions {
+        isCoreLibraryDesugaringEnabled = true
+    }
```

Comment:
why do we need that? If I'm not wrong both Single FIT and Sample apps are running on API higher than 24? or you want to make them compatible if they are executed on emulator with Android below N?

Reply:
Exactly that, just to make sure.